src/main/java/eutros/multiblocktweaker/crafttweaker/construction/MultiblockBuilder.java

```diff
@@ -121,6 +127,49 @@ public MultiblockBuilder withZoom(float zoom) {
         return this;
     }
 
+    /**
+     * Add a tooltip to a specific {@link net.minecraft.item.ItemStack} in a Multiblock Structure.
+     * Multiple tooltips can be added to the same {@link net.minecraft.item.ItemStack}.
+     * This is optional, and if not defined will not add any tooltips.
+     *
+     * @param itemStack - The ItemStack form of the block to add the tooltip to.
+     * @param tooltip - The localization key for the tooltip to be added.
+     * @return This builder, for convenience.
+     */
+    @ZenMethod
+    public MultiblockBuilder withPartTooltip(IItemStack itemStack, crafttweaker.api.text.ITextComponent tooltip) {
+
+        ItemStack actualStack = CraftTweakerMC.getItemStack(itemStack);
+        ITextComponent tooltipText = CraftTweakerMC.getITextComponent(tooltip);
+
+        List<ITextComponent> possibleTooltips = tooltipMap.getOrDefault(actualStack, null);
+
+        if(possibleTooltips == null) {
+            List<net.minecraft.util.text.ITextComponent> tooltipToAdd = new ArrayList();
+            tooltipToAdd.add(tooltipText);
+            this.tooltipMap.put(actualStack, tooltipToAdd);
+        } else {
+            possibleTooltips.add(tooltipText);
+            this.tooltipMap.put(actualStack, possibleTooltips);
+        }
+
+        return this;
+    }
+
+    /**
+     * A Helper method to clear pre-existing tooltips from Multiblocks.
+     * This should always be called before tooltips are added.
```

Comment:
Is this line correct? This method doesn't ever need to be called, and its ordering doesn't matter, so this may be misleading.